src/pages/IssuerCreate.tsx

```
@@ -0,0 +1,635 @@
+
+	const handlePaymentMethodChange = (
+		index: number,
+		field: 'paymentMethodType' | 'discountRate',
+		value: string | number,
+	) => {
+		setPaymentMethods((prev) =>
+			prev.map((item, i) => (i === index ? { ...item, [field]: value } : item)),
+		);
+	};
+
+	const handleImageUpload = async (info: any, isDescription = false) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)